### PR TITLE
Update CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by nitya**
Added section for #Challenge25 contributors (and placeholder for missing sections)

### CODE REVIEW

**Comment by johnpapa** [comment]
From all of us on the Cloud Advocacy team at Microsoft, thank you for creating this pull request!


Have a Happy Holiday season!


We'll not review Issues or PRs in this period of time.

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -441,3 +447,19 @@ We will be listing contributors in alphabetical order, by week, by challenge. At
...
+
+---
+
+#### WEEK FOUR - CHALLENGE 24
+
+---
+
+#### WEEK FOUR - CHALLENGE 25
+
+- [Barbara 4bes](https://github.com/Ba4bes/25daysofserverless/tree/master/Daya25KheerRecipe) - `PowerShell function App, Cognitive services (SpeechToText), Storage Account`
```
Comment (simonaco) [inline CONTRIBUTORS.md]: Can you update to include Barbara's last name? Barbara Forbes

Comment (nitya) [reply CONTRIBUTORS.md]: Changes made.

**Comment by johnpapa** [comment]
test

**Pull request merged**